src/libhirte/common/cfg.c

```diff
@@ -155,12 +162,20 @@ int cfg_load_from_dir(struct config *config, const char *custom_config_directory
         number_of_files = scandir(
                         custom_config_directory, &all_files_in_dir, &is_file_name_ending_with_conf, alphasort);
         if (number_of_files < 0) {
+                // consider a missing confd directory as no error
+                if (errno == ENOENT) {
+                        return 0;
+                }
```

Comment:
Just a thought: wouldn't it be more clear to run access before scandir, so we don't need to enter scandir if the directory doesn't exist?

Reply:
Good point, added the access check.